common/djangoapps/student/forms.py

```diff
@@ -234,3 +237,33 @@ def cleaned_extended_profile(self):
             for key, value in self.cleaned_data.items()
             if key in self.extended_profile_fields and value is not None
         }
+
+
+RegDropDownContext = namedtuple(
+    'RegDropDownContext', ['dropdown_visibility', 'dropdown_label', 'dropdown_missing_error', 'dropdown_choices']
+)
+
+
+def registration_dropdown_context():
+    """
+    Returns a dictionary containing context values for the dropdown menu for use in a template or
+    other situations which require fallback defaults.
+    """
+    return RegDropDownContext(
+        getattr(settings, 'REGISTRATION_EXTRA_FIELDS', {}).get('dropdown', 'hidden'),
+        getattr(settings, 'REGISTRATION_DROPDOWN_LABEL', gettext('How did you hear of us?')),
+        getattr(
+            settings,
+            'REGISTRATION_DROPDOWN_MISSING_ERROR',
+            gettext('You must select how you heard of us.')
+        ),
+        getattr(
+            settings,
+            'REGISTRATION_DROPDOWN_CHOICES',
+            (
+                ('search', gettext('Search Engine')),
+                ('friend', gettext('Referred by a Friend')),
+                ('other', gettext('Other')),
+            )
```

Comment:
?? Why are you explicitly providing a fallback? If there's nothing specified in settings, I would expect nothing new to be added.